schemas/groups/resources.json

```diff
@@ -220,6 +220,22 @@
               "type": "string",
               "description": "Location of the note"
             },
+            "group": {
+                "type": "string",
+                "description": "Name of a group or collection to which the note is attached"
+            }, 
+            "author": {
```

Comment:
Here I would copy the `contributors` structure from package.json:
- `authors` to allow multiple
- either objects with name/email/url properties (anyOf) or strings with suggested structure

https://flaviocopes.com/package-json/#contributors

Even if the current use case is note per author I have a hunch allowing multiple would be useful.

Reply:
Updated property name to `authors` and aligned structure to package.json `contributors`.